balancer/weightedroundrobin/balancer_test.go

```diff
@@ -732,7 +791,7 @@ func ensureReached(ctx context.Context, t *testing.T, c testgrpc.TestServiceClie
 	reached := make(map[string]struct{})
 	for len(reached) != n {
 		var peer peer.Peer
-		if _, err := c.EmptyCall(ctx, &testpb.Empty{}, grpc.Peer(&peer)); err != nil {
+		if _, err := c.EmptyCall(ctx, &testpb.Empty{}, grpc.Peer(&peer), grpc.WaitForReady(true)); err != nil {
```

Comment:
Why?  Are we expecting connection failures?

Reply:
Whoops I had this left over from when I was trying to fix what I solved in the other PR with the endpoint hierarchy. Removed.